### ISSUE TITLE
Fix misleading security permissions

### ISSUE THREAD

**Issue by tractorcow** [silverstripe/silverstripe-framework#4861]
When permissions are managed via the SiteConfig, certain permissions applicable to a currently edited group are not apparent.

We should probably move some of the global permissions, perhaps as a readonly segment, into the group permissions dialog.

I.e. move some of this:

![image](https://cloud.githubusercontent.com/assets/936064/11941914/af587ad8-a897-11e5-8a2e-6caf4fa9cbd7.png)

Onto this dialog:

![image](https://cloud.githubusercontent.com/assets/936064/11941922/c241bbb4-a897-11e5-9420-7219ad3449e5.png)

**Comment by jonom** [comment]
#### Security settings

I think augmenting that list with some read-only permissions sounds like a great idea - perhaps under a label like 'Default content permissions', with a note about the fact they're due to settings in the Access section. Something like: *"You can manage default content access permissions for groups under 'Security > Access'. These may be overridden on the Settings tab of individual pages."*

Then I think renaming the existing 'Content permissions' to 'Super content permissions' or similar would be helpful and sit nicely alongside this new read-only 'Default content permissions' section.

I also think we should move the Access tab from SiteConfig to Security. SiteConfig is supposed to be an optional module and Security is a more logical home for these settings anyway I think.

#### Individual page settings

Suggest that as part of this we revise the permissions dialog for individual pages too - currently you find this under Settings even for root pages:

![screen shot 2015-12-21 at 1 51 16 pm]
(https://cloud.githubusercontent.com/assets/1079425/11942232/19294b8a-a7ea-11e5-91dc-f87d8d2f676c.png)
Which doesn't make sense for root pages as there is no parent page. In that case it actually means "Use site config settings" currently.

I'd suggest that both of these field groups should be changed to only show two options initially: "Inherit" and "Custom". If you choose the Custom option then the specific options are revealed so you can make a selection, but they're hidden otherwise. Additionally the Inherit option should indicate what the inherited settings are and ideally where they are coming from, with a link or maybe just a tooltip:

![settings](https://cloud.githubusercontent.com/assets/1079425/11942931/fbca2b40-a7ee-11e5-8057-f9eefb1ba43e.png)

**Comment by tractorcow** [comment]
Yeah, I think we need to ensure those permissions bubble up to the individual site trees as well; Good spotting. :)

I think the concept is that at every point, the user should be adequately informed about:
- What parent permissions are in effect
- When those permissions are overridden by the current option
- When those permissions are inherited by the current option
- The effect of any inherited permissions + selected options

**Comment by tractorcow** [comment]
SiteConfig is mandatory for CMS module, which is where those permissions are inheriting from, so it's possibly ok to leave it there.

It's probably ok to add these to the security section, but it would probably need to involve quite a bit of UX re-think of that entire section. I don't think it's mandatory for this issue.

**Comment by clarkepaul** [comment]
Related https://github.com/silverstripe/silverstripe-framework/issues/3310

**Comment by newleeland** [comment]
This may be resolved in #8620 together with other CMS permission issues as a holistic solution
